skins/standard/global.css

```diff
@@ -35,7 +35,8 @@
     }
     /* monospace is much smaller than Verdana by default, so we make it a bit bigger. */
     pre, code, kbd {
-        font-size: medium;
+        font-family: "Droid Sans Mono", Menlo, Monaco, "Courier New", Courier, monospace;
```

Comment:
Does this set of changes bring the User Story font-size and font-family into alignment with the other instances of our use of Droid?